frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideOutBranchAction.java

```diff
@@ -34,6 +41,8 @@
       IBranchNameProvider,
       IExpectsKeyGitMacheteRepository {
 
+  public static final String SLIDE_OUT_DELETION_SUGGESTION_REMEMBERED = "git-machete.slide-out.deletion-suggestion.shown";
```

Comment:
`shown` vs `REMEMBERED`... although I'd say it's neither, it's more sth like `SHOULD_SHOW`?